_episodes/01-basics.md

```diff
@@ -28,41 +28,40 @@ Word's [Track Changes](https://support.office.com/en-us/article/Track-changes-in
 history](https://support.google.com/docs/answer/190843?hl=en), or LibreOffice's [Recording and Displaying Changes](https://help.libreoffice.org/Common/Recording_and_Displaying_Changes).
 
 Version control systems start with a base version of the document and
-then save just the changes you made at each step of the way. You can
-think of it as a tape: if you rewind the tape and start at the base
-document, then you can play back each change and end up with your
-latest version.
+then record changes you make each step of the way. You can
+think of it as a recording of your progress: you can rewind to start at the base
+document, play back each change you made, eventually arriving at your
```

Comment:
I think this is a good change, since of course tapes are much less common now unless you are serious about long-time off-site data preservation. However, I would recommend replacing the comma after "document" with the word "and".

Reply:
Weighing in as a maintainer, this change looks good to me. I too agree with the addition of "and" after the use of "document". Maybe you could update your PR with this minor change @caseyyoungflesh?

Reply:
Agreed. Will make that change